WebDriverAgentLib/Utilities/FBConfiguration.h

```diff
@@ -19,6 +19,9 @@ NS_ASSUME_NONNULL_BEGIN
 /*! If set to YES will ask TestManagerDaemon for element visibility */
 @property (class, nonatomic, assign) BOOL shouldUseTestManagerForVisibilityDetection;
 
+/* The maximum typing frequency for all typing activities */
```

Comment:
This might be max chars count per second, but yeah, I cannot get any info on this as well.